subprojects/docs/src/docs/userguide/gradleWrapper.adoc

```diff
@@ -170,9 +170,9 @@ Use the Gradle `wrapper` task to generate the wrapper, specifying a version. The
 ++++
 
 [[customizing_wrapper]]
-=== Customizing the Gradle Wrapper
+=== Customizing the Gradle Wrapper Task
```

Comment:
For subtitles we only capitalize the first word. The words "Gradle" and "Wrapper" are capitalized because they refer to a concrete product/feature. In this title "Task" should start with a lower case letter.